homeassistant/components/omnilogic/omnilogic_common.py

```diff
@@ -0,0 +1,119 @@
+"""Common classes and elements for Omnilogic Integration."""
```

Comment:
We can rename this module to just `common.py`.

Reply:
Addressed